device/cuda/src/utils/make_prefix_sum_buff.cu

```diff
@@ -51,4 +52,28 @@ vecmem::data::vector_buffer<device::prefix_sum_element_t> make_prefix_sum_buff(
     return prefix_sum_buff;
 }
 
+vecmem::data::vector_buffer<device::prefix_sum_element_t> make_prefix_sum_buff(
+    const std::vector<device::prefix_sum_size_t>& sizes, vecmem::copy& copy,
+    const traccc::memory_resource& mr, const stream& str) {
+
+    const device::prefix_sum_buffer_t make_sum_result =
+        device::make_prefix_sum_buffer(sizes, copy, mr);
+    const vecmem::data::vector_view<const device::prefix_sum_size_t>
+        sizes_sum_view = make_sum_result.view;
+    const unsigned int totalSize = make_sum_result.totalSize;
+
+    // Create buffer and view objects for prefix sum vector
+    vecmem::data::vector_buffer<device::prefix_sum_element_t> prefix_sum_buff(
+        totalSize, mr.main);
+    copy.setup(prefix_sum_buff);
+
+    // Fill the prefix sum vector
+    kernels::fill_prefix_sum<<<(sizes_sum_view.size() / 32) + 1, 32, 0,
```

Comment:
This is the incorrect way to compute grid sizes.

Reply:
Was just a dumb copy-paste on my part.